lib/ansible/modules/notification/snow_record.py

```diff
@@ -326,6 +381,20 @@ def run_module():
     module.exit_json(**result)
 
 
+def updater(new_token):
+    print("OAuth token refreshed!")
```

Comment:
`print` statements are not used while Ansible run, you may want to use `module.warn` instead.